internet-shop/src/core/services/productService.ts

```diff
@@ -0,0 +1,12 @@
+import { ProductState } from '../redux/types/products/productType';
+import axiosInstance from '../../config/axios';
+import PRODUCT_URL from '../constants/urlConstants';
+
+export class ProductService {
+  public static async GetAll(): Promise<ProductState[]> {
+    const result = await axiosInstance.get<ProductState[]>(PRODUCT_URL)
+      .then((data) => data.data)
+      .catch(({ response }) => console.log(response.data));
```

Comment:
Also I think you can create some function somewhere to use in this kind of situations line ({response}) => expeptionTracker(response) and in this function u can write console.log or some other outputs, and then when you no longer what to use console.log but some other form of output you need to change just on func instead of every catch case